src/components/darkmode-switch/types.ts

```diff
@@ -0,0 +1,2 @@
+export type Theme = 'system-default' | 'force-light' | 'force-dark';
+export const THEME_EVENT_NAME = 'kompendiumTheme';
```

Comment:
Is `types.ts` a good name for this file? I mean, `Theme` is a type, but `THEME_EVENT_NAME` isn't… I can't come up with a better name though…